docs/reference-manual/native-image/JFR.md

```diff
@@ -115,16 +110,98 @@ Otherwise, this option expects a comma separated list of tag combinations, each
+
+
+### Event Streaming
+(Since GraalVM for JDK 17 / 20).
+Event streaming is supported in Native Image. Event Streaming lets you register callbacks for specific events at the application level. This introduces more flexibility and control over how recordings are managed. For example, you may dynamically increase the duration threshold of an event if it is found in the stream beyond a certain number times. Event streaming also allows the application to get continuous periodic JFR updates that are useful for monitoring purposes. 
+
+Currently, stacktraces are not supported on streamed events. This means you cannot access the stacktrace of an event inside its callback method. However, this limitation does not affect stacktraces in the JFR snapshot (.jfr) file, those will still work as usual.   
+
+### Interaction with FlightRecorderMXBean via Remote JMX
+(Since GraalVM for JDK 17 / 20).
+You can now interact with Native Image JFR from out of process via remote JMX connection to `FlightRecorderMXBean`.  This can be done in using applications such as JDK Mission Control or VisualVM. Over JMX you can start, stop, and dump JFR recordings using the `FlightRecorderMXBean` API as an interface. [This blog post](https://developers.redhat.com/articles/2023/06/13/improvements-native-image-jfr-support-graalvm-jdk-20?source=sso#new_supported_features) provides a step-by-step walk-through of how to start using JFR over remote JMX in JDK Mission Control.
+Find more general information on how to use JMX with Native Image in [this guide](guides/build-and-run-native-executable-with-remote-jmx.md).
+
+> Note: Remote JMX support in Native Image is experimental and may be changed in the future. 
+
+### Event Throttling
+(Build from source).
+Similar to OpenJDK, event throttling is enabled by default for `jdk.ObjectAllocationSample`. This limits the overhead of the event. The unsampled version of this event is `jdk.ObjectAllocationInNewTLAB`. You can specify whether to enable this event as well as the throttling rate within a JFR configuration file (.jfc). 
+
+### FlightRecorderOptions 
+(Build from source).
+You can fine tune JFR parameters by using `-XX:FlightRecorderOptions` at runtime. This is primarily for advanced users, and most people should be fine with the default parameters. The options are the same as in OpenJDK.
+
+### Leak Profiling
+(Build from source).
+Leak profiling implemented by the `jdk.OldObjectSample` event is partially implemented. Specifically, old object tracking is possible, but path to GC root information is unavailable. 
+
+### Built-In Events
+Many of the VM-level built-in events found in OpenJDK are available in Native Image. Java-level events implemented by bytecode instrumentation in the Java HotSpot VM are not yet supported in Native Image. 
```

Comment:
If we have to choose either the website documentation or the GitHub issue, it might be better to choose the website documentation because it lives in the same place as the actual code. 

Additionally, it will help support others more easily automatically parse what events are supported. Here's an example: https://github.com/parttimenerd/jfreventcollector/issues/7

Reply:
@roberttoyonaga, as you may already know, the GraalVM docs are versioned. The best way would be to keep a table listing all events that are supported in a release but without specifying the GraalVM version. When the new events are added in the upcoming release, we just add the items to the table. So just removing the second column is fine.

I can also add a table listing events supported in GraalVM for JDK 17 and JDK 21 and backport those with the next CPU release.